test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,8 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+  it("returns a reversed array of the number's digits", () => {
+    expect(numberToReversedDigits(12345)).toEqual([5, 4, 3, 2, 1]);
+    expect(numberToReversedDigits(582)).toEqual([2,8,5])
```

Comment:
Could add a test here for edge cases, like a single digit number being passed in, or if feeling more adventurous how would you would a negative number (if at all)?